oeps/architectural-decisions/oep-0014-proc-archive-repos.rst

```diff
@@ -27,31 +27,34 @@ OEP-14: Archiving edX GitHub Repositories
 
 .. _ORA PR Discussion: https://github.com/edx/edx-ora/pull/187
 .. _Initial Archiving Discussions: https://openedx.atlassian.net/wiki/display/IT/Proposed+Github+Deprecation+Process
+
+**Code Review: This link is private. Could someone move it to the COMM space or suggest a better alternative?**
```

Comment:
Code reviewers: Looking for suggestions here

Reply:
I've requested to move the page (https://openedx.atlassian.net/servicedesk/customer/portal/1/ITSUPPORT-17691 though you won't be able to see that ticket).  It has four proposals for how to deprecate repos, maybe we don't need the history?

Reply:
Yeah, since I can't see the page I'm not sure what the intention was of linking it. Perhaps it was simply having the history of considered and rejected alternatives?

Reply:
I think having history is nice in general, but it doesn't have to block this PR from merging.